tests/platform/cloudfoundry/test_local_discovery.py

```
@@ -0,0 +1,84 @@
+    project_path = os.getenv(constants.PROJECT_PATH)
+    cf_files_path = os.getenv(constants.CLOUDFOUNDRY_FILES_PATH)
+
+    if not project_path:
+        raise Exception("PROJECT_PATH environment variable is not set")
+    if not cf_files_path:
+        raise Exception("CLOUDFOUNDRY_FILES_PATH environment variable is not set")
+
+    # Path to the local manifest file
+    manifest_file = os.path.join(os.getenv(constants.PROJECT_PATH), 'data', 'yaml', 'platform', 'cloudfoundry', 'hello-spring-cloud-manifest.yml')
+    if not os.path.exists(manifest_file):
+        raise Exception(f"Manifest file not found: {manifest_file}")
```

Ruff (0.15.0)

[warning] Create your own exception

(TRY002)